Description: fix test output for Node.js ≥ 12
Author: Xavier Guimard <yadd@debian.org>
Forwarded: no
Last-Update: 2020-06-22

--- a/test/log.js
+++ b/test/log.js
@@ -19,7 +19,7 @@
 
     util.log(1, 2, 3, 4, 'five');
     var time = util.date(new Date(), 'HH:MM:ss');
-    writtenValue.should.eql('[' + util.colors.grey(time) + '] 1 2 3 4 \'five\'\n');
+    writtenValue.should.eql('[' + util.colors.grey(time) + '] 1 2 3 4 five\n');
 
     done();
   });
